sbe-tool/src/main/java/uk/co/real_logic/sbe/generation/cpp/CppGenerator.java

```diff
@@ -1299,6 +1299,14 @@ private static CharSequence generateFixedFlyweightCode(final String className, f
             "    static SBE_CONSTEXPR const std::uint64_t encodedLength(void)\n" +
             "    {\n" +
             "        return %2$s;\n" +
+            "    }\n\n" +
+            "    std::uint64_t offset(void) const\n" +
```

Comment:
I don't think so -- SBE uses `uint64_t` internally for offsets everywhere (this is wrong, but that's another story).